functions.php

```diff
@@ -133,7 +133,7 @@ function twentytwenty_theme_support() {
 		 * by the theme.
 		 */
 		$loader = new TwentyTwenty_Script_Loader();
-		add_filter( 'script_loader_tag', [ $loader, 'filter_script_loader_tag' ], 10, 2 );
+		add_filter( 'script_loader_tag', array( $loader => 'filter_script_loader_tag' ), 10, 2 );
```

Comment:
This should still remain in the format of `array($obj, 'myCallbackMethod')`.

```suggestion
		add_filter( 'script_loader_tag', array( $loader, 'filter_script_loader_tag' ), 10, 2 );
```

Reply:
Updated: https://github.com/WordPress/twentytwenty/pull/484/commits/72a427983ab8ae36d988d6b59898b6423a9550e8